synopsys-task/src/main.ts

```diff
@@ -19,7 +19,9 @@ import {
 export async function run() {
   console.log("Synopsys Task started...");
   const tempDir = getTempDir();
+  taskLib.debug("tempDir:::" + tempDir);
```

Comment:
can we just remove this logging(too much colons) and do it normally:
taskLib.debug(`tempDir: ${tempDir}`);
better to use template literals..try to apply and correct logging statements in all cases

Reply:
Updated everywhere as per your suggestion